index.html

```diff
@@ -895,34 +727,20 @@ <h3 class="bannerbox__title">Newsletter</h3>
             });
 
             // Submit Function for needRide (Rider) form.
-            $('form[name="needRide"]').submit(function() {
+            $('form[name="needRide"]').on('submit', function() {
                 document['needRide'].action = "https://api.carpoolvote.com/v1.1/rider";
```

Comment:
Someone else has mentioned that. Just seeing them there, I thought they must provide a handy thing to override when debugging, but it is duplication